styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+  font-family: 'Roboto', Helvetica, sans-serif;
+}
+
+body {
+  background-color: #191765;
+  max-width: 375px;
+  background-image: url(images/Ellipse\ 18.svg);
+  background-repeat: no-repeat;
+  background-position: 0 47px;
+}
```

Comment:
Please kindly set max-width: 100% this will make your page more responsive Also this will get rid of the blank space after to hamburger icon.

Your design: 

![Screenshot from 2022-04-21 09-39-56](https://user-images.githubusercontent.com/79506105/164387851-d0d86341-9920-4f8d-a80b-ee03860fe6d8.png)
How Page will look after you set max-width: 100%:
![Screenshot from 2022-04-21 09-51-42](https://user-images.githubusercontent.com/79506105/164388060-317b94ca-7a08-4a75-95c0-e45d0e430121.png)